Defaulting to user installation because normal site-packages is not writeable
Requirement already up-to-date: pip in /apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages (20.2.3)
Collecting package metadata (current_repodata.json): ...working... failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/akode/linux-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https://conda.anaconda.org/akode/linux-64'


Traceback (most recent call last):
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages/gym/envs/atari/atari_env.py", line 9, in <module>
    import atari_py
ModuleNotFoundError: No module named 'atari_py'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "Type_1.py", line 154, in <module>
    main()
  File "Type_1.py", line 36, in main
    env = gym.make("Bowling-v0").unwrapped 
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages/gym/envs/registration.py", line 142, in make
    return registry.make(id, **kwargs)
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages/gym/envs/registration.py", line 87, in make
    env = spec.make(**kwargs)
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages/gym/envs/registration.py", line 58, in make
    cls = load(self.entry_point)
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages/gym/envs/registration.py", line 17, in load
    mod = importlib.import_module(mod_name)
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages/gym/envs/atari/__init__.py", line 1, in <module>
    from gym.envs.atari.atari_env import AtariEnv
  File "/apps/pkg/pytorch/1.6.0/rhel7_u5/cuda10.2/lib/python3.7/site-packages/gym/envs/atari/atari_env.py", line 13, in <module>
    "'pip install gym[atari]'.)".format(e))
gym.error.DependencyNotInstalled: No module named 'atari_py'. (HINT: you can install Atari dependencies by running 'pip install gym[atari]'.)
